### PR TITLE
ERS-31031: Whole lot of documentation cleanup and tying up of loose ends

### PR DESCRIPTION
**Pull request by RoseWyatt**
Associated PR:
https://github.com/SAP-docs/preview.developer.concur.com/pull/1283

Changes:
* MANAGER context support properly documented for a variety of endpoints
* `Update submitted report` endpoint had its name, description, and schema updated to better cater to external callers. Concept of 'Short url vs long url' removed from any naming conventions. Description updated to include limitations around when this URL can be used.
* `Get reports to approve`endpoint got a once over with the big change being adding in a proper response schema on top of the example given.
* Comments API updated with the proper employee schema. We do not return `first name`/`last name`/ `middle initial`, we return an `employeeUuid` and `employeeId`

### CODE REVIEW

File: src/api-reference/expense/expense-report/v4.reports.markdown [hunk 1]
```diff
@@ -612,152 +604,188 @@ curl --location --request GET https://integration.api.concursolutions.com/expens
...
+| `id`      | `string` | -      | The ID of the validation error.               |
+| `message` | `string` | -      | The detailed message of the validation error. |
+| `source`  | `string` | -      | The type of validation which failed.          |
+
+### <a name="employee-schema"></a>Employee
+
+| Name           | Type     | Format | Description                                                                        |
+|----------------|----------|--------|------------------------------------------------------------------------------------|
+| `employeeId`   | `string` | -      | Unique identifier of the employee that can be referenced in external systems.      |
+| `employeeUuid` | `string` | -      | UUID of the employee used to identify the employee within internal concur systems. |
```
Comment (RoseWyatt) [inline src/api-reference/expense/expense-report/v4.reports.markdown:791]: Updated, correct Employee object.

File: src/api-reference/expense/expense-report/v4.reports.markdown [hunk 2]
```diff
@@ -612,152 +604,188 @@ curl --location --request GET https://integration.api.concursolutions.com/expens
...
+| `reportDate`              | `string`                                | `YYYY-MM-DD`   | The date assigned to the report by the user.                                                                                                                                                                        |
+| `reportSource`            | `string`                                | -              | **Required** The source of the report. Supported values: `EA` - Expense Assistant, `MOB` - Mobile, `OTHER` - Unknown, `SE` - Smart Expense, `TR` - Travel Request, `UI` - Web UI                                    |
+| `startDate`               | `string`                                | `YYYY-MM-DD`   | The start date of the report as used for trip-based reporting.                                                                                                                                                      |
+
+### <a name="update-submitted-report-schema"></a>Update Submitted Report
```
Comment (RoseWyatt) [inline src/api-reference/expense/expense-report/v4.reports.markdown:720]: submitted report update schema

File: src/api-reference/expense/expense-report/v4.reports.markdown [hunk 3]
```diff
@@ -612,152 +604,188 @@ curl --location --request GET https://integration.api.concursolutions.com/expens
...
+| `totalApprovedAmount` | [`Amount`](#amount-schema)     | -                      | The total approved amount for the report                                                                                                                                          |
+| `hasExceptions`       | `boolean`                      | `true`/`false`         | Whether the Report has any exceptions.                                                                                                                                            |
+| `reportType`          | `string`                       | -                      | This value identifies the method used to create the report. Statement refers to company billed statement reports and auto-created refers to reports created by Expense Assistant. |
+| `links`               | [`Link`](#link-schema)         | -                      | Resource links related to this call.                                                                                                                                              |
+
```
Comment (RoseWyatt) [inline src/api-reference/expense/expense-report/v4.reports.markdown:625]: new report to approve schema.

File: src/api-reference/expense/expense-report/v4.reports.markdown [hunk 4]
```diff
@@ -577,30 +593,6 @@ curl --location --request GET https://integration.api.concursolutions.com/expens
...
-  ],
-  "customResponseData": {
-    "additionalProp1": "string",
-    "additionalProp2": "string",
-    "additionalProp3": "string"
-  },
-  "path": "string"
-}
-```
-
```
Comment (RoseWyatt) [inline src/api-reference/expense/expense-report/v4.reports.markdown:603]: Removed this exception response, I put this here wayyy back when I first did this externalization and its unnecessary.

File: src/api-reference/expense/expense-report/v4.reports.markdown [hunk 5]
```diff
@@ -392,9 +397,13 @@ curl --location --request PATCH 'https://us.api.concursolutions.com/expenserepor
...
 
-## Update a Submitted Report <a name="update-submitted-report"></a>
+## Update Submitted Report <a name="update-submitted-report"></a>
 
-Updates the report header in an unsubmitted or submitted expense report specified and is restricted to modify `Business Purpose`, `Comment`, `Custom/Org unit`, `Name` and `Paper Receipt Received` fields only (only Company JWT Allowed).
+Updates the report header in an unsubmitted or submitted expense report. When calling this endpoint only Company JWT
+authentication is allowed. Since this endpoint supports updating submitted reports and does not require a user and
+context, only a limited set of fields can be updated through it.
+
+A submitted report cannot be updated even through this endpoint once it has reached a 'Paid' workflow status.
```
Comment (RoseWyatt) [inline src/api-reference/expense/expense-report/v4.reports.markdown:406]: Updated description.

File: src/api-reference/expense/expense-report/v4.allocations.markdown [hunk 6]
```diff
@@ -25,22 +27,24 @@ Access to this documentation does not provide access to the API.
 
 Required Scopes:
 
-|Name|Description|Endpoint|
-|---|---|---|
-|`expense.report.read`|Get information about expense reports.|GET|
-|`expense.report.readwrite`|Read and write expense report headers.        |PATCH|
-|`user.read`|Get User Information, necessary for `userID`.|GET|
+| Name                       | Description                                   | Endpoint |
+|----------------------------|-----------------------------------------------|----------|
```
Comment (RoseWyatt) [inline src/api-reference/expense/expense-report/v4.allocations.markdown:31]: Every single table was reformatted to line up perfectly, makes viewing/editing the actual markdown file much easier.

**Review by RoAnnC1 — APPROVED** [review]
:shipit:

File: src/api-reference/expense/expense-report/v4.exceptions.markdown [hunk 7]
```diff
@@ -54,13 +56,12 @@ https://{datacenterURI}/expensereports/v4/users/{userID}/context/{contextType}/r
...
-|`reportId`|`string`|-|**Required** The unique identifier of the report that is being read.|
-|`excludeExpenses`|`boolean`|true or false|Include only exceptions for the report header. Default value: false|
-
+| Name              | Type      | Format        | Description                                                                                                                                                   |
+|-------------------|-----------|---------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `userID`          | `string`  | -             | **Required** The unique identifier of the SAP Concur user. Use [Identity v4](/api-reference/profile/v4.identity.html) to retrieve the `userID`.               |
+| `contextType`     | `string`  | -             | **Required** The access level of the SAP Concur user, which determines the form fields they can view/modify. Supported values: `TRAVELER`, `MANAGER`, `PROXY` |
```
Comment (RoseWyatt) [inline src/api-reference/expense/expense-report/v4.exceptions.markdown:62]: MANAGER is now documented as a supported context for all exception endpoints.

**Review by RoAnnC1 — APPROVED** [review]
:shipit:

**Pull request merged**